### PR TITLE
SSL enabled database connection for workflow repository (#1712)

### PR DESCRIPTION
**Pull request by AntoineDao**
Quick fix to enable SSL connection to Postgres databases for workflow repositories.

**disclaimer**: couldn't get this to work mysql as there didn't seem to be an option to do so using`upper.io/db.v3`...

**disclaimer 2**: I'm a bit of a noob when it comes to Golang so didn't manage to run tests/write new ones locally. I'm totally counting on your CI system to flag my possible mistakes :smile:

### CODE REVIEW

**Comment by sarabala1979** [comment]
This PR will resolve  #1712

**Comment by AntoineDao** [comment]
Anything I can do on my end to facilitate this moving into master? Looking to implement this feature on our end and wanting to avoid diverging forks as much as possible :smile:

File: workflow/persist/sqldb/sqldb.go [hunk 1]
```diff
@@ -85,6 +85,14 @@ func CreatePostGresDBSession(kubectlConfig kubernetes.Interface, namespace strin
 		Host:     postgresConfig.Host + ":" + postgresConfig.Port,
 		Database: postgresConfig.Database,
 	}
+
+	if postgresConfig.SSL {
+		var options := map[string]string{
+			"sslmode": "true"
+		}
+		settings.Options = options
+	}
```
Comment (simster7) [inline workflow/persist/sqldb/sqldb.go]: This code has language errors and won't compile. Please replace with:

```go
	if postgresConfig.SSL {
		options := map[string]string{
			"sslmode": "true",
		}
		settings.Options = options
	}
```

Comment (sarabala1979) [reply workflow/persist/sqldb/sqldb.go]: @AntoineDao! Please fix @simster7 comments. PR looks good.

**Comment by logicfox** [comment]
This might also resolve #1564 temporarily, until a fix for that issue is identified

File: workflow/persist/sqldb/sqldb.go (hunk 1 above)
Comment (AntoineDao) [reply workflow/persist/sqldb/sqldb.go]: Man I'm a bit embarrassed I didn't pick that up :see_no_evil:... I'll up my Go game before my next PR I promise!

**Pull request merged**